firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -147,6 +147,7 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
     @MainActor
     func testHistoryTableContextMenu() {
         navigator.openURL(loremIpsumURL)
+        waitUntilPageLoad()
```

Comment:
This test fails intermittently. If the page isn't loaded fully, the tabs button does not work.